src/test/velodrome.test.ts

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Dhedge } from "..";
+import { Network } from "../types";
+import { TEST_POOL, WSTETH, WETH } from "./constants";
+import { getTxOptions } from "./txOptions";
+
+import { wallet } from "./wallet";
+
+const wETHwstETHLp = "0xBf205335De602ac38244F112d712ab04CB59A498";
+// const wETHwstETHGauge = "0x131Ae347E654248671Afc885F0767cB605C065d7";
+
+let dhedge: Dhedge;
+let options: any;
+jest.setTimeout(100000);
+
+describe("pool", () => {
+  beforeAll(async () => {
+    dhedge = new Dhedge(wallet, Network.OPTIMISM);
+    options = await getTxOptions(Network.OPTIMISM);
+  });
+
+  // it("approves unlimited WETH on for Velodrome", async () => {
+  //   let result;
+  //   const pool = await dhedge.loadPool(TEST_POOL);
+  //   try {
+  //     result = await pool.approve(
+  //       Dapp.VELODROME,
+  //       WSTETH,
+  //       ethers.constants.MaxInt256,
+  //       options
+  //     );
+  //     console.log(result);
+  //   } catch (e) {
+  //     console.log(e);
+  //   }
+  //   expect(result).not.toBe(null);
+  // });
+
+  // it("adds WETH and wstETH to a Velodrome stable pool", async () => {
+  //   let result;
+  //   const pool = await dhedge.loadPool(TEST_POOL);
+  //   const wethBalance = await dhedge.utils.getBalance(WETH, pool.address);
+  //   const stwethBalance = await dhedge.utils.getBalance(WSTETH, pool.address);
+
+  //   try {
+  //     result = await pool.addLiquidityVelodrome(
+  //       WETH,
+  //       WSTETH,
+  //       wethBalance,
+  //       stwethBalance,
+  //       true,
+  //       options
+  //     );
+  //     console.log(result);
+  //   } catch (e) {
+  //     console.log(e);
+  //   }
+  //   expect(result).not.toBe(null);
```

Comment:
I don't feel strongly (as it's pretty straight forward), but given you get the LP balance of the pool in the next test,
You should probably check that the balance is > 0 in this test (just to ensure it does add liquidity properly)

Could improve the `expect` checks on each test eg
- getting LP balance
- getting Gauge token balance, etc